contracts/voting-verifier/src/state.rs

```diff
@@ -19,6 +19,8 @@ pub struct Config {
     pub block_expiry: u64, // number of blocks after which a poll expires
     pub confirmation_height: u64,
     pub source_chain: ChainName,
+    pub rewards_contract: Addr,
+    pub grace_period: u64, // TODO: add update mechanism to change this after instantiation
```

Comment:
Do we want to just use the block_expiry as the grace period? All votes that are cast before expiration count towards rewards, and all votes after do not?